scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml

```diff
@@ -29,11 +29,21 @@
 
 - name: Download Intel SGX DCAP Driver
   get_url:
-    url: "{{ intel_sgx_driver_url }}"
+    url: "{{intel_sgx_w_flc_driver_url}}"
     dest: /tmp/sgx_linux_x64_driver_dcap.bin
     mode: 0755
     timeout: 120
   retries: 3
+  when: flc_enabled|bool
+
+- name: Download Intel SGX1 Driver
+  get_url:
+    url: "{{intel_sgx1_driver_url}}"
+    dest: /tmp/sgx_linux_x64_driver_dcap.bin
```

Comment:
Do we want to be storing both the sgx1 and dcap driver under the same destination?

If this is the case, a recommendation would be to abstract this out and to use a variable.

If this is not the case, a recommendation could be to refactor.

Reply:
@johnkord agree about having a generic name. Pushing a commit to fix this.